deployment.rb

```diff
@@ -187,6 +187,18 @@ def hourofcode_hostname
     canonical_hostname('hourofcode.com')
   end
 
+  def circle_run_identifier
+    ENV['CIRCLE_BUILD_NUM'] ? "CIRCLE-BUILD-#{ENV['CIRCLE_BUILD_NUM']}-#{ENV['CIRCLE_NODE_INDEX']}" : nil
```

Comment:
@davidsbailey I think we'll also want to use this single-point-of-truth helper in circle.rake when spinning up the Sauce Connect tunnel (which connect.rb connects to in turn): https://github.com/code-dot-org/code-dot-org/blob/e47fc281aa1c2013da2b24678726b259f87dbc3e/lib/rake/circle.rake#L31-L31